.travis.yml

```diff
@@ -14,20 +14,22 @@ compiler:
 
 install:
     # TODO should we not use only from from openblas or just atlas?
-    - sudo apt-get update && sudo apt-get install -y build-essential libpng12-dev libfreetype6-dev libopenblas-dev libopenblas-base liblapack-dev gfortran  git python python-dev python-pip libsqlite3-dev wget libatlas-base-dev &> /dev/null
+    - sudo apt-get update && sudo apt-get install -y build-essential libpng12-dev libfreetype6-dev libopenblas-dev libopenblas-base liblapack-dev gfortran  git python python-dev python-pip libsqlite3-dev wget libatlas-base-dev zip &> /dev/null
     - echo "/usr/lib/atlas-base" | sudo tee /etc/ld.so.conf.d/atlas-lib.conf && sudo ldconfig  # Hack: solution from http://unix.stackexchange.com/questions/52703/using-atlas-from-scipy
     - (pip install -r alex-requirements.txt 2>&1 > /tmp/pip_log.log & PIP_PID=$! ; while [ "$(ps -p ${PIP_PID} | wc -l)" -eq "2" ]; do sleep 60; echo "pip is still running"; done;)
     - echo "#####################"; echo "vvv PIP LOG vvv"; tail -n 100 /tmp/pip_log.log ; echo "#####################";
     # Install pjsip.
     - (git clone https://github.com/UFAL-DSG/pjsip.git && pushd pjsip && git checkout travis && ./configure CXXFLAGS=-fPIC CFLAGS=-fPIC LDFLAGS=-fPIC CPPFLAGS=-fPIC && make dep && make && sudo make install && cd pjsip-apps/src/python/ && pip install . && popd && python -c 'import pjsuaxt') &> /dev/null
     # Unpack and install precomplied openfst, pykaldi and pyfst
-    - (wget https://s3-eu-west-1.amazonaws.com/ufal-dsg/pykaldi/pykaldi-travis_2.7.tar.gz/pykaldi/pykaldi_Ubuntu_12.04_LTS.tar.gz && tar xf pykaldi_Ubuntu_12.04_LTS.tar.gz && pushd pykaldi_Ubuntu_12.04_LTS && sudo cp -r openfst/lib/* /usr/local/lib && sudo ldconfig && sudo tar xf pyfst*.tar -C / && sudo tar xf pykaldi*.tar -C / ; popd ) &> /dev/null
+    - wget https://s3-eu-west-1.amazonaws.com/ufal-dsg/pykaldi/pykaldi-travis_2.7.tar.gz/pykaldi_Ubuntu_12.04_LTS.zip && unzip pykaldi_Ubuntu_12.04_LTS.zip && pushd pykaldi_Ubuntu_12.04_LTS && sudo cp -r openfst/lib/* /usr/local/lib && easy_install pyfst*.egg && easy_install pykaldi*.egg && sudo ldconfig && popd
```

Comment:
When you keep the parentheses ( ) around the command you can leave out the trickery with pushd and popd =)